src/openklant/components/klantinteracties/api/tests/test_notifications.py

```diff
@@ -0,0 +1,295 @@
+from unittest.mock import patch
+
+from django.test import override_settings
+
+from freezegun import freeze_time
+from notifications_api_common.models import NotificationsConfig
+from rest_framework import status
+from vng_api_common.tests import reverse
+from zgw_consumers.constants import APITypes
+from zgw_consumers.models import Service
+
+from openklant.components.klantinteracties.models.tests.factories.actoren import (
+    ActorFactory,
+)
+from openklant.components.klantinteracties.models.tests.factories.digitaal_adres import (
+    DigitaalAdresFactory,
+)
+from openklant.components.klantinteracties.models.tests.factories.internetaken import (
+    InterneTaakFactory,
+)
+from openklant.components.klantinteracties.models.tests.factories.klantcontacten import (
+    KlantcontactFactory,
+)
+from openklant.components.klantinteracties.models.tests.factories.partijen import (
+    PartijFactory,
+)
+from openklant.components.klantinteracties.models.tests.factories.rekeningnummer import (
+    RekeningnummerFactory,
+)
```

Comment:
I think if you import these factories in `openklant/components/klantinteracties/models/tests/factories/__init__.py` you can all import them from `.factories`